### ISSUE TITLE
OPS-0012 Composition_Openstax icon issue in POET editor

### ISSUE THREAD

**Issue by srm-openstax** [openstax/cnx-recipes#4071]
The Openstax icon is not visible in the POET editor. So, Toc editor and push content options could not work. I have attached the screenshot for your reference. Kindly check and resolve this issue.

![poet-toc-issue](https://user-images.githubusercontent.com/86622862/130753473-caadd254-8c83-4483-8b5f-e217f9102241.png)

# Bug Location(s)
server: https://copper-anteater-egyq5vxq.ws-us16.gitpod.io/
Location: Poet editor/Toolbar

**Comment by ne-newton** [comment]
In the extensions section, an extra extension `Auto Markdown TOC` had been installed, and the OpenStax `editor` plugin was not installed. I loaded the Gitpod workspace and re-installed the `editor` plugin.
I don't know if the `editor` plugin was manually uninstalled at the same time the `Auto Markdown TOC` plugin was manually installed. Having the OpenStax `editor` plugin not install is not a bug we have encountered in testing. Typically, the extensions should be left alone though, and extra plugins should not be installed.
